examples/advanced/broadcasting_your_own_methods.py

```diff
@@ -1,14 +1,16 @@
 """Broadcast functions across multi-dimensional data
 =====================================================
 
-Use the ``make_broadcastable`` decorator to efficiently
-apply functions across any data dimension.
+Use the :func:`movement.utils.broadcasting.make_broadcastable` decorator
+(:func:`@make_broadcastable()<movement.utils.broadcasting.make_broadcastable>`)
+to efficiently apply functions across any data dimension.
```

Comment:
I propose removing the links from this sentence only, because this text gets rendered as is when hovering over the example card. The preview you get that way is unreadable.

I thinks it's fine to do this because there are other links later in the example.

```suggestion
Use the @make_broadcastable decorator
to efficiently apply functions across any data dimension.
```